webpack-test/ConfigTestCases.template.js

```diff
@@ -1,6 +1,6 @@
 "use strict";
 
-require("./helpers/warmup-webpack");
+// require("./helpers/warmup-webpack");
```

Comment:
Do we need to warmup? cause our core is implemented by rust

Reply:
Also noticed that the `warmup` is used for fix azure pipeline, which we don't have.